src/components/Links/Links.styles.ts

```diff
@@ -11,7 +11,7 @@ export const LinksContainer = styled.main`
   padding-bottom: 10%;
 `;
 
-export const Link = styled.a<LinkProps>`
+export const Link = styled.a.attrs({ target: "_blank" })<LinkProps>`
```

Comment:
This will help prevent setting target to `_blank` inside the component's props.

Check if any element sets the prop **target** to `_blank`